include/irr/video/IGPUObjectFromAssetConverter.h

```diff
@@ -397,7 +399,7 @@ auto IGPUObjectFromAssetConverter::create(asset::ICPUBufferView** const _begin,
 auto IGPUObjectFromAssetConverter::create(asset::ICPUDescriptorSetLayout** const _begin, asset::ICPUDescriptorSetLayout** const _end) -> created_gpu_object_array<asset::ICPUDescriptorSetLayout>
 {
     const auto assetCount = std::distance(_begin, _end);
-    auto res = core::make_refctd_dynamic_array<created_gpu_object_array<asset::ICPUDescriptorSet> >(assetCount);
+    auto res = core::make_refctd_dynamic_array<created_gpu_object_array<asset::ICPUDescriptorSetLayout> >(assetCount);
 
     core::vector<asset::ICPUSampler*> cpuSamplers;//immutable samplers
     size_t maxSamplers = 0ull;
```

Comment:
we not only need to eliminate duplicates by pointer, but also duplicates by content.

Reply:
nah we never did this, might be an extension TODO for eliminateDuplicatea func

Reply:
because there was never any point to compare contents XD